app/Nova/User.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
Probably we could but then we'd need to register all resources separately.

Reply:
It can be changed using:
```
protected function resources()
{
    Nova::resourcesIn(app_path('Nova'));
}
```
https://nova.laravel.com/docs/3.0/resources/#registering-resources

Reply:
In this case I would rather register them one by one. At least it will be non magical thing.

Reply:
It doesn't look to bad or magical when you look into `resourcesIn` implementation:
```
    public static function resourcesIn($directory)
    {
        $namespace = app()->getNamespace();

        $resources = [];

        foreach ((new Finder)->in($directory)->files() as $resource) {
            $resource = $namespace.str_replace(
                ['/', '.php'],
                ['\\', ''],
                Str::after($resource->getPathname(), app_path().DIRECTORY_SEPARATOR)
            );

            if (is_subclass_of($resource, Resource::class) &&
                ! (new ReflectionClass($resource))->isAbstract() &&
                ! (is_subclass_of($resource, ActionResource::class))) {
                $resources[] = $resource;
            }
        }

        static::resources(
            collect($resources)->sort()->all()
        );
    }
```

It does exactly the same thing in base `vendor/laravel/nova/src/NovaApplicationServiceProvider.php`:
```
    protected function resources()
    {
        Nova::resourcesIn(app_path('Nova'));
    }
```

Reply:
I used `Nova::resources()` approach.